pkgs/development/python-modules/ratarmountcore/default.nix

```
@@ -15,19 +15,15 @@
 
 buildPythonPackage rec {
   pname = "ratarmountcore";
-  version = "0.6.0";
+  version = "0.6.3";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and add setuptools to nativeBuildInputs.